platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/crypto/CryptoStatic.java

```diff
@@ -541,6 +541,7 @@ public static SecureRandom getNonDetRandom() {
      * 		the current configuration
      * @return a map of KeysAndCerts objects, one for each node
      */
+    @SuppressWarnings("java:S2142")
```

Comment:
Can you please add a comment on which warning we are suppressing here?

Reply:
Just spoke with @cody-littley. He proposed we will not add any suppress on that class just mark it as not fix on the Sonar side.